tests/bagatom/test_nodeToXML.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from datetime import datetime
+from lxml import etree
+
+from codalib import bagatom
+
+
+class NodeStub(object):
+    """
+    Test stub that implements the Node model interface.
+    """
+    node_name = "Test Name"
+    node_url = "http://example.com/node"
+    node_path = "/foo/bar/node"
+    node_capacity = 4096
+    node_size = 2048
+    last_checked = datetime(2015, 01, 01)
+
+
+def test_return_type():
+    """
+    Verify the return type of nodeToXML is an instance of
+    etree._Element.
+    """
+    tree = bagatom.nodeToXML(NodeStub())
+    assert isinstance(tree, etree._Element)
+
+
+@pytest.mark.parametrize("element,text", [
```

Comment:
I know the pytest examples show double quotes, but they use them consistently, whereas you have double quotes here and single quotes in the list. Do you have to use double quotes here?